common/metrics/defs.go

```diff
@@ -1965,6 +1967,7 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		TaskLimitExceededCounter:                          {metricName: "task_errors_limit_exceeded_counter", metricType: Counter},
 		TaskProcessingLatency:                             {metricName: "task_latency_processing", metricType: Timer},
 		TaskQueueLatency:                                  {metricName: "task_latency_queue", metricType: Timer},
+		StuckTaskDiscardedCounter:                         {metricName: "stuck_task_discarded_counter", metricType: Counter},
```

Comment:
I would be specific for this like "transfer_task_missing_event_counter"

Reply:
Done.